biomage/rotate_ci/rotate_ci.py

```diff
@@ -259,12 +252,7 @@ def rotate_ci(token, org, repo):
 
     g = Github(token)
     org = g.get_organization(org)
-    repos = []
-
-    if repo == "all":
-        repos = org.get_repos()
-    else:
-        repos = [org.get_repo(repo)]
+    repos = org.get_repos()
```

Comment:
Just get all the repos under the same organization. This removes the `-r` command and makes the command simpler.